service/account/handlers_accounts_test.go

```diff
@@ -153,25 +153,31 @@ func (s *AccountSuite) TestCreateGetUpdateAccountHandlers(c *check.C) {
 
 	account := datastore.Account{ID: 2, AuthorityID: "vendor", ResellerAPI: false}
 	acc, _ := json.Marshal(account)
+	accBrokenJSON := []byte(`{"user"`)
 
 	tests := []AccountTest{
```

Comment:
This is not especially easy or pleasant to review and I'm surprised by the sheer number of changes here, including ones that don't seem obviously relevant

Thoughts on splitting it up into single-change commits / adding some more clarity in some other way?

Reply:
I added some missing test cases, but you are right, I will remove all not necessary test cases from this PR (but yes, the way the tests are structured makes it hard to fix/review)

Reply:
Thanks! Definitely agree that these tests _generally_ aren't easy to review, but it's much clearer now you've dropped the extraneous ones cheers!